code/modules/shuttle/marine_dropship.dm

```diff
@@ -968,19 +974,20 @@
 		to_chat(user, "<span class='warning'>Access Denied!</span>")
 		return
 	var/list/options = valid_destinations()
-	var/obj/docking_port/mobile/M = SSshuttle.getShuttle(shuttleId)
+	var/obj/docking_port/mobile/marine_dropship/M = SSshuttle.getShuttle(shuttleId)
 	var/dat = "Status: [M ? M.getStatusText() : "*Missing*"]<br><br>"
 	if(M)
-		var/destination_found
-		for(var/obj/docking_port/stationary/S in SSshuttle.stationary)
-			if(!options.Find(S.id))
-				continue
-			if(!M.check_dock(S, silent=TRUE))
-				continue
-			destination_found = TRUE
-			dat += "<A href='?src=[REF(src)];move=[S.id]'>Send to [S.name]</A><br>"
-		if(!destination_found)
-			dat += "<B>Shuttle Locked</B><br>"
+		if (M.hijack_state==HIJACK_STATE_NORMAL)
```

Comment:
?.
and lacking spaces

Reply:
The console at CIC still proposed destination even when alamo was hijacked, even though clicking on the buttons didn't do anything. It is now more consistent